trunk/source/Library-UltimateCore/src/de/uni_freiburg/informatik/ultimate/core/lib/results/InvariantResult.java

```diff
@@ -79,4 +79,8 @@ public String getLongDescription() {
 	public Set<Check> getChecks() {
 		return mChecks;
 	}
+	
+	public boolean isLoopLocation() {

```

Comment:
I think it makes more sense to call this `isLoopInvariant()`.

Reply:
Yes that sounds nicer, thanks!

Reply:
Done in e391017